### DISCUSSION TITLE
Document alternative status codes/possible errors per path?

### DISCUSSION THREAD

**Discussion by kristianvld** [fastapi/fastapi#8262] [Questions] [answered]
**Description**

Is it possible to document errors and other possible status codes for a path, like the 422 Validation Error that is automatically documented? Can't find anything about documenting errors in the FastAPI docs. Even the "Handling Errors" section only describes how to throw HTTPErrors, but not how to document them.

**Comment by tiangolo** [comment]
Not yet, there will be a way to document other responses including errors. I'm on it right now :smile:

**Comment by tiangolo** [comment]
Here it is, you can now add a `responses` parameter with additional responses, including other status codes, media types, etc.

It also allows extending the main response.

https://fastapi.tiangolo.com/tutorial/additional-responses/

**Comment by kristianvld** [comment]
Sweet man, seems to wok nicely 👌

**Comment by tiangolo** [comment]
Great! Thanks for reporting back and closing the issue.

**Comment by tfpgh** [comment, marked as answer]
> Here it is, you can now add a `responses` parameter with additional responses, including other status codes, media types, etc.
> 
> It also allows extending the main response.
> 
> https://fastapi.tiangolo.com/tutorial/additional-responses/

Just a note, this page is now in the Advanced User Guide at [https://fastapi.tiangolo.com/advanced/additional-responses/](https://fastapi.tiangolo.com/advanced/additional-responses/)
